pkgs/development/mobile/genymotion/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
   name = "genymotion-${version}";
```

Comment:
And this could be changed to:
```nix
pname = "genymotion";
```
Note: The version is not appended to pname and the name will be built from pname and version.

Reply:
Thanks for the suggestion. I have applied this change.